conf/evolutions/collins/15.sql

```diff
@@ -0,0 +1,13 @@
+# --- Add GPU information to asset_meta
```

Comment:
👀  Mind updating this to match what it does? :)

Reply:
:man_facepalming: 

Good catch.